src/journey-maps/package.json

```diff
@@ -14,7 +14,7 @@
     "@angular/common": "0.0.0-NG",
     "@angular/core": "0.0.0-NG",
     "@types/geojson": ">=7946",
-    "maplibre-gl": "^2.1.0"
+    "maplibre-gl": "3.6.2"
```

Comment:
Is this supposed to be an exact version and not a version range?

Reply:
Done.